druid-shell/src/platform/x11/window.rs

```diff
@@ -536,19 +487,17 @@ impl Window {
             key_code,
             key_code,
         );
-        match self.handler.try_borrow_mut() {
-            Ok(mut handler) => {
-                handler.key_down(key_event);
-                Ok(())
-            }
-            Err(err) => Err(Error::BorrowError(format!(
-                "Window::handle_key_press handle: {}",
-                err
-            ))),
-        }
+        self.handler
+            .try_borrow_mut()
+            .context("Window::handle_key_press")?
```

Comment:
Lets keep the variable name in the context message. (`handler` instead of `handle` though)